node/hack/governor/src/index.ts

```diff
@@ -160,12 +215,43 @@ axios
               notional +
               "\n";
 
-            //console.log("chain: " + chain + ", addr: " + data.Address + ", symbol: " + data.Symbol + ", notional: " + notional + ", price: " + data.TokenPrice + ", amount: " + data.Amount)
+            newTokenKeys[chain + "-" + wormholeAddr + "-" + data.Symbol] = true;
+            newTokensCount += 1;
           }
         }
       }
     }
 
+    for (var token of existingTokenKeys) {
+      // A token has been removed from the token list 
+      if (!newTokenKeys[token]) {
+        removedTokens.push(token);
+      }
+    }
+
+    // Sanity check to make sure the script is doing what we think it is
+    if (existingTokenKeys.length + addedTokens.length - removedTokens.length != newTokensCount) {
+      console.error("The new number of tokens doesn't make sense");
+      process.exit(1);
+    }
+
+    var changedContent = "**Tokens before** = " + existingTokenKeys.length;
+    changedContent += "\n**Tokens after** = " + newTokensCount;
+    changedContent += "\n\n**Tokens added** = " + addedTokens.length + ":\n*<WH_chain_id>-<WH_token_addr>-<token_symbol>*\n\n";
+    changedContent += JSON.stringify(addedTokens, null, 1);
+    changedContent += "\n\n**Tokens removed** = " + removedTokens.length + ":\n*<WH_chain_id>-<WH_token_addr>-<token_symbol>*\n\n";
+    changedContent += JSON.stringify(removedTokens, null, 1);
+    changedContent += "\n\n**Tokens with significant price drops (>" + PriceDeltaTolerance + "%)** = " + significantPriceChanges.length + ":\n\n"
+    changedContent += JSON.stringify(significantPriceChanges, null, 1);
```

Comment:
Please wrap `changes.txt` contents in triple backticks to make it a preformatted code block in the PR body. It would read nicer that way.

```suggestion
    changedContent += JSON.stringify(significantPriceChanges, null, 1);
    changedContent += "```";
```

Reply:
Agreed it looks nicer as a code block rather than formatted text, have updated